docs/docs/add-a-manifest-file.md

```diff
@@ -1,32 +1,32 @@
+Son sitios web corrientes que aprovechan la funcionalidad de los navegadores web modernos para mejorar la experiencia web con características y beneficios similares a las aplicaciones. Visita [la visión general de Google](https://developers.google.com/web/progressive-web-apps/) que define una experiencia PWA y [la documentación sobre Aplicaciones Web Progresivas (PWAs)](/docs/progressive-web-app/) para aprender como un sitio Gatsby es una Aplicación Web Progresiva.
 
-The inclusion of a web app manifest is one of the three generally accepted [baseline requirements for a PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
+La inclusión de un archivo de manifiesto de aplicación web, es uno de los tres [requisitos base para una PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1) generalmente aceptados.
 
-Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
+Citando a [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
 
-> The web app manifest is a simple JSON file that tells the browser about your web application and how it should behave when 'installed' on the user's mobile device or desktop.
+> El archivo de manifiesto de aplicación web es un archivo JSON simple que informa al navegador acerca de su aplicación web y como se debe comportar cuando sea 'instalada' en el dispositivo móvil o escritorio del usuario.
 
-[Gatsby's manifest plugin](/packages/gatsby-plugin-manifest/) configures Gatsby to create a `manifest.webmanifest` file on every site build.
+[El plugin de manifiesto de Gatsby](/packages/gatsby-plugin-manifest/) configura Gatsby para crear un archivo `manifest.webmanifest` en cada sitio construido.
 
-### Using `gatsby-plugin-manifest`
+### Usando `gatsby-plugin-manifest`
 
-1.  Install the plugin:
+1.  Instala el plugin:
 
 ```shell
 npm install --save gatsby-plugin-manifest
 ```
 
-2. Add a favicon for your app under `src/images/icon.png`. The icon is necessary to build all images for the manifest. For more information look at the docs of [`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
+2. Añade un 'favicon' a tu aplicación en `src/images/icon.png`. El icono es necesario para construir todas las imágenes para el archivo de manifiesto. Para más información, visita la documentación en [`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
 
-3. Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+3. Añade el plugin al array de `plugins` en tu fichero `gatsby-config.js`.
```

Comment:
Following the [Glossary](https://github.com/gatsbyjs/gatsby-es/blob/master/TRANSLATION.md#traducciones-comunes), array should be _array_ (`_array_` ).

Reply:
Changed! Thanks!